test/torch/tensors/test_poly.py

```diff
@@ -0,0 +1,102 @@
+from syft.frameworks.torch.tensors.interpreters.Polynomial import PolynomialTensor
+import torch
+
+
+""" Test cases to ensure working of Polynomial Tensor. The tests under these ensure that the error between actual funcion values and approximations do not deviate too much"""
+
+# Maximum permissible error as calculated by EvalRelative under PolynomialTensor
+
+DATA_THRESHOLD = 5.5
+ERROR_THRESHOLD = 5
+DATA_SIZE = 50000
+
+
+def EvalError(x_true, x_pred):
+
+    """ The function is used to measure the error between actual function value and approximated function value. It is evaluated as follows.
+        Given a tensor of 1000 random numbers, the function approximation of individual numbers in the tensor must be within atleast +- 5% error of actual value.
+        This threshold is given by ERROR_THRESHOLD.
+        But , a certain percentage of numbers in the tensor can exceed the ERROR_THRESHOLD. This threshold is described by DATA_THRESHOLD.
+        The function returns true the total data error (% of individual approximations which have error within ERROR_THRESHOLD)
+        
+            Parameters: 
+            
+            x_true: Value of true function 
+            x_pred: Value of function approximation 
+            
+            Returns:
+                
+            data_error: 
+            
+        """
+
+    # Relative absolute error give by abs(true_value-prediction_value)/(true_value)
+    error_rel = torch.div(torch.abs(x_true - x_pred), x_true) * 100
+
+    # Check if individual approximations are within the threshold
+    error = error_rel < ERROR_THRESHOLD
+
+    count = 0
+
+    for i in error:
+
+        if i.item() == 0:
+            count += 1
+
+    data_error = (count / len(x_true)) * 100
+
+    return data_error
+
+
+def test_EvalError():
+
+    " Ensure that EvalRelative function does what it has to. For the purpose we take a tensor of ones and subtract and add an amount of error "
+
+    one = torch.ones(6)
+    two = torch.ones(6) * 2
+
+    assert EvalError(two, one) == 100
+    assert EvalError(one, one) == 0
+    assert EvalError(two, two) == 0
+
+    three = torch.tensor([1.0, 1.0, 1.0, 0.0, 0.0, 0.0])
+
+    assert EvalError(three, one) == 50
+    assert EvalError(one, three) == 50
+
+
+def testSigmoid():
+
+    Ptensor = PolynomialTensor()
+
+    x = torch.randn(DATA_SIZE)
+
+    m = torch.nn.Sigmoid()
```

Comment:
Don't compare performance to nn.sigmoid, compare it to what the value of the approximation should be. We don't care how good the polynomial approximation is in terms of accuracy, we care how accurately it is implemented